### PR TITLE
Ensure component updates when value is changed elsewhere

### PR DESCRIPTION
**Pull request by shackpank**
The `defaultValue` attribute will not re-render inputs if it is changed, `value` will.

I've also added a code coverage command for Travis to run.

### CODE REVIEW

**Comment by robhuzzey** [comment]
+1

File: .istanbul.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+instrumentation:
+    extensions: ['.js', '.jsx']
```
Comment (hxpaul) [inline .istanbul.yml]: you could fix this spacing here

File: .travis.yml [hunk 2]
```diff
@@ -4,7 +4,7 @@ node_js:
   - "0.12"
   - "0.10"
 script:
-  - npm test
```
Comment (hxpaul) [inline .travis.yml:7]: @hx-markterry suggested in my pr these should go in a `npm run ci` script in package.json

**Comment by hxpaul** [comment]
my npm was way out of date, resulting in some funnies on npm install, so scratch that last comment!

**Comment by hxpaul** [comment]
:+1:

**Pull request merged**